controllers/release/release_adapter.go

```diff
@@ -160,6 +165,37 @@ func (a *Adapter) EnsureReleasePipelineStatusIsTracked() (results.OperationResul
 	return results.ContinueProcessing()
 }
 
+// EnsureSnapshotEnvironmentBindingIsCreated is an operation that will ensure that a SnapshotEnvironmentBinding is created
+// or updated for the current Release.
+func (a *Adapter) EnsureSnapshotEnvironmentBindingIsCreated() (results.OperationResult, error) {
```

Comment:
It is a little confusing to me that we use the full CRD name sometimes and not others. I know the CRD name is long, but that isn't our fault. Can we use the full CRD name until it changes?

Reply:
Yep, discussed offline. This will be the final name of the resource so in those cases where it was possible, it's already renamed.